tb_plugin/torch_tb_profiler/profiler/overall_parser.py

```diff
@@ -2,120 +2,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # --------------------------------------------------------------------------
 import sys
-from enum import IntEnum
 
 from .. import utils
 from .trace import EventTypes
+from .range_lib import *
 
 logger = utils.get_logger()
 
 
-def merge_ranges(src_ranges, is_sorted=False):
-    merged_ranges = []
-    if len(src_ranges) > 0:
```

Comment:
I already move these logic to a separated file, please merge from plugin/0.2

Reply:
Yes, merged.